OpenRA.Mods.Common/Projectiles/AreaBeam.cs

```diff
@@ -28,7 +28,7 @@ public class AreaBeamInfo : IProjectileInfo
 		public readonly WDist[] Speed = { new WDist(128) };
 
 		[Desc("The maximum duration (in ticks) of each beam burst.")]
-		public readonly int Duration = 10;
+		public readonly int BeamDuration = 10;
```

Comment:
I intend to add `DamageDuration` in my next PR and felt this would be better to separate them, but if you prefer to rename `BeamDuration` in `LaserZap` to just `Duration` instead, I guess I can do that.